assets/meta.yaml

```diff
@@ -793,3 +793,27 @@
   prohibited_uses: ''
   monitoring: none
   feedback: none
+
+- type: model
+  name: Llama 3
+  organization: Meta
+  description: Llama 3 is the third generation of Meta AI's open-source large language model. It comes with pretrained and instruction-fine-tuned language models with 8B and 70B parameters that can support a broad range of use cases.
+  created_date: 2024-04-18
+  url: https://llama.meta.com/llama3/
+  model_card: https://github.com/meta-llama/llama3/blob/main/MODEL_CARD.md
+  modality: text; text
+  analysis: The models were evaluated based on their performance on standard benchmarks and real-world scenarios. These evaluations were performed using a high-quality human evaluation set containing 1,800 prompts covering multiple use cases. The models also went through red-teaming for safety, where human experts and automated methods were used to generate adversarial prompts to test for problematic responses.
+  size: 70B parameters
```

Comment:
we should make sure to update this when llama 3 400b comes out

Reply:
yup!